front/src/dashboard/components/BSDList/BSDasri/WorkflowAction/PublishBsdasri.tsx

```diff
@@ -52,6 +54,9 @@ export default function PublishBsdasri({ form }: WorkflowActionProps) {
             tableau de bord transporteur. Le bordereau pourra toujours être
             modifié ou supprimé depuis l'onglet "Suivi".
           </p>
+          <p className="tw-mt-1">
+            Le statut du bordereau passera de "brouillon" à "initial"
```

Comment:
Le message me paraît un peu technique, j'aurai suggéré :

```suggestion
            Une fois publié, le bordereau sera prêt à l'enlèvement
```